src/build_occurrence_based_indicators.Rmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Occurrence-based indicators of Invasive Alien Species"
+author:
+- Damiano Oldoni
+- Add your name
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    toc: true
+    toc_depth: 3
+    toc_float: true
+    number_sections: true
+    theme: yeti
+    df_print: paged
+knit: (function(input_file, encoding) { rmarkdown::render(input_file, encoding = encoding, output_file = paste0("../docs/",sub(".Rmd", ".html", basename(input_file))))})
+---
+
+This document describes how to build indicators of Invasive Alien Species based on occurrence data.
+The following indicators will be built:
```

Comment:
I would make clear that this is a file that will collect the other ones and add the downloading code section in this document only, not in the other ones (keep it DRY).